gamemodes/terrortown/gamemode/shared/lang/fr.lua

```diff
@@ -525,7 +525,7 @@ Appuyez sur {menukey} pour recevoir votre équipement !]]
 L.info_popup_traitor_alone = [[
 Vous êtes un TRAITRE ! Vous n'avez pas d'amis traitres ce round.
 
-Tuez tous les autres pour gagner !
+Tuez tous le monde pour gagner !
```

Comment:
Is the space in front of the exclamation mark intentional? Same goes for the next line

Reply:
Yes its intentional but i can remove it if needed

Reply:
At least in German and in English there is no space in front of a punctuation mark. I don't know how it is in French though.

Reply:
![Capture d’écran 2021-01-06 112810](https://user-images.githubusercontent.com/73431903/103758699-d5b56380-5012-11eb-82ea-40b2e561b931.png)
with space 
![Capture d’écranedededede 2021-01-06 113149](https://user-images.githubusercontent.com/73431903/103758710-db12ae00-5012-11eb-81e8-2bfaa9161cab.png)
without